pkgs/applications/window-managers/i3/blocks.nix

```
@@ -13,6 +13,14 @@ stdenv.mkDerivation {
     sha256 = "1fx4230lmqa5rpzph68dwnpcjfaaqv5gfkradcr85hd1z8d1qp1b";
   };
 
+  patches = [
+    (fetchpatch {
```

```suggestion
    # XDG_CONFIG_DIRS can contain multiple elements separated by colons, which should be searched in order.
    (fetchpatch {
```

so I can know clearly from looking at the nix expression why we have this patch in the future, or for the maintainiers.